### ISSUE TITLE
Verify cleanup branch with R Studio before merge

### ISSUE THREAD

**Issue by trevorcampbell** [UBC-DSCI/introduction-to-datascience#14]
Need to check the `cleanup` branch with R Studio to make sure I haven't removed any critical files before we merge into `master`. It works fine when compiling the book from the REPL.

@ttimbers could you give it a shot?

**Comment by ttimbers** [comment]
OK, I just pulled in the `cleanup` branch on my local, and built the book using `bookdown::render_book("index.Rmd", "bookdown::gitbook")` and locally it builds fine. However I cannot see the [`.nojekyll`](https://github.com/UBC-DSCI/introduction-to-datascience/blob/master/docs/.nojekyll) file, and we need that (in the `docs` directory) to tell GitHub that our website is not to be built via Jekyll, since the bookdown HTML output is already a standalone website (see [`bookdown` docs on deploying using GitHub](https://bookdown.org/yihui/bookdown/github.html)). Not sure if other files are needed as well, that is the only one I can think of... 

I guess we can just merge and see what happens, we can always revert back if needed.

**Comment by trevorcampbell** [comment]
I did notice that, but on my own personal site I don't have a `.nojeykll` file and it works fine. I did a bit of snooping as to why this is required and it looks like it's because [jekyll removes filenames that start with an underscore](https://github.blog/2009-12-29-bypassing-jekyll-on-github-pages/) -- but none of the bookdown published files have the underscore. Anyway I'll merge without it, give it a test, and add the .nojekyll file back if things break!
